infrastructure/staging/efs.tf

```diff
@@ -13,3 +13,19 @@ module "efs" {
 
   efs_access_security_group_ids = [aws_security_group.efs.id]
 }
+
+module "efs-workernode" {
+  source = "../modules/efs"
+
+  name = "workflow-stage-workernode-storage"
+
+  throughput_mode                 = "provisioned"
+  provisioned_throughput_in_mibps = "5"
```

Comment:
I think we should start without provisioned mode and see how performance turns out. This adds extra cost and can easily be modified later.